.github/workflows/cd-pipeline.yaml

```diff
@@ -1,4 +1,4 @@
-name: 'VIB Publish'
+name: 'CD Pipeline'
```

Comment:
`CI Pipeline` just to avoid confusions. Technically we are releasing.

Reply:
After talking about this privately, it is true both pipelines have steps related to integration.

I thought over this and having both pipelines named as CI (even using different filenames) could be confusing. Another option would be to use CT (continuous testing) and CI to keep them differentiated but we are also automatically merging PRs in `vib-verify`.

After looking over the options and the pipelines definitions, I think keeping them as CI/CD is appropriate, even if not 100% accurate. We do merge PRs into master in `vib-verify` so it can be considered continuous integration (CI) and we publish the new chart's version to the Helm repository in `vib-publish`, which can be considered continuous delivery (CD).

This can be a bit subjective so I do not have strong thoughts on this and can be left to a majority vote among reviewers 😄

Reply:
In my opinion, we should keep differentiating between the pipeline that publishes and the one that only tests the changes/merge the PR.